index.html

```diff
@@ -1,252 +1,248 @@
-                        </ul>
-                    </div>
-                    <button type="button" class="get">Get my resume</button>
-                    <ul>
+                    <a class="connect">LET'S CONNECT</a>
+                    <ul class="social">
                         <li>
-                            <div class="languages-skills">
-                                <img src="img/Frame 7.png">
-                            </div>
+                            <a href="#">
+                                <img src="img/Vector4.png" alt="github picture">
+                            </a>
                         </li>
                         <li>
-                            <div class="my-skills">
-                                <img src="img/Frame 9.png" alt="javascript picture">
-                                <img src="img/Frame 10.png" alt="html picture">
-                                <img src="img/Frame 11.png" alt="css picture">
-                            </div>
+                            <a href="#">
+                                <img src="img/Vector3.png" alt="linked picture">
+                            </a>
                         </li>
                         <li>
-                            <div class="languages-skills">
-                                <img src="img/Frame 8.png">
-                            </div>
+                            <a href="#">
+                                <img src="img/Vector5.png" alt="peace symbol">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="img/Vector.png" alt="twitter picture">
+                            </a>
                         </li>
                         <li>
-                            <div class="languages-skills">
-                                <img src="img/Frame 9 (1).png">
-                            </div>
+                            <a href="#">
+                                <img src="img/Vector1.png" alt="Gmail picture">
+                            </a>
                         </li>
                     </ul>
-                </section>
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+                <div class="tool">
+                    <button class="language-button">Languages <span class="fa fa-chevron-down"></span></button>
+                    <ul class="languages">
+                        <li>
+                            <a href="#"><i class="fa-brands fa-html5"></i>html</a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa-brands fa-css3-alt"></i>css</a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa-brands fa-js"></i>javascript</a>
+                        </li>
+                    </ul>
```

Comment:
- Kindly make sure you use appropriate languages icone you can import the ones used in the Figma design as images and style them. your design must stick as much as possible to the Figma design :+1: 

## Your design 


![image](https://user-images.githubusercontent.com/95297251/221250552-6143fca4-f567-417f-aab2-4957aa5994ba.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/221251342-df3de2c3-e56f-42e3-8829-3a0b3c638eff.png)